client/src/components/LoginBtn.js

```diff
@@ -29,7 +41,11 @@ function LoginBtn(props) {
       onSuccess={login}
       onFailure={handleLoginFailure}
       cookiePolicy={"single_host_origin"}
-      responseType="code,token"
+      scope="https://www.googleapis.com/auth/calendar.events"
+      responseType="code"
+      accessType="offline"
+      // prompt='consent'
```

Comment:
remove these if not used

Reply:
the scope is for enabling our access to google calendar
responsetype + accesstype is to make google return authorization code, instead of access token